test/arrays/ferguson/semantic-examples/valgrind-futures/4-return-alias-local.chpl

```diff
@@ -1,7 +1,7 @@
 use samples;
 proc f() {
   var A:[1..4] R;
-  var B => A;
+  ref B = A;
```

Comment:
I didn't put any significant thought into this edit (other than wondering what "slice" its comments were referring to) and was just looking for tests that needed updating w.r.t. `=>`, doing the obvious rewrite, and then seeing if they still worked.  I hadn't yet gotten around to seeing whether this would still fail under valgrind, but testing today, see that it now passes.

I  don't have a problem with removing it, but also don't see a downside to retaining it (promoting it to its parent directory?) as proof that we don't get memory errors for returning a `ref` alias to a local array.  Your test, though, so your call.